### PR TITLE
chore: improve release docs

### PR DESCRIPTION
**Pull request by cjihrig**
This commit fixes a typo and the test command used in deno_std.

### CODE REVIEW

File: tools/cut_a_release.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ Before starting the process write a message in company's #general channel:
       1. Ensure the version in `version.ts` is updated correctly.
       2. Ensure `Releases.md` is updated correctly.
       3. Ensure all the tests pass with the latest build
-         - Use `../deno/target/release/deno test -A --unstable` and NOT
-           `deno task test`
+         - Use
+           `../deno/target/release/deno test --doc --unstable --allow-all --ignore=node/`
```
Comment (dsherret) [inline tools/cut_a_release.md:47]: Even though it's a feat, we could maybe consider including https://github.com/denoland/deno/tree/66b9187a2dd2ea96e760b9bcd81a578dd338147d in the 1.21.2 release if it's not too much trouble (we usually just include unstable features in patch releases). Then we could change this to `../deno/target/release dneo task test`

**Review by dsherret — APPROVED** [review]
LGTM

**Review by piscisaureus — APPROVED** [review]

File: tools/cut_a_release.md (hunk 1 above)
Comment (cjihrig) [reply tools/cut_a_release.md:47]: @dsherret the release took a long time today, so I didn't want to go back and start over. Do you want me to:

1. Land this as is.
2. Use your suggested text and that commit can go out in the next release.
3. Leave this open until that commit is actually released.

Comment (dsherret) [reply tools/cut_a_release.md:47]: Makes sense! Yeah, landing this as-is is good.

**Pull request merged**